pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -101,7 +101,10 @@ usage_percentage=$(( (disk_usage * 100) / disk_space ))
 echo "[disk usage checker] Volume usage percentage: current = ${usage_percentage} ; allowed = ${usage_percentage_limit}"
 if [ "$usage_percentage" -gt "$usage_percentage_limit" ]; then
 	echo "[disk usage checker] Disk usage exceeds the volume percentage of ${usage_percentage_limit} for mounted directory, terminating..."
-	pkill --signal SIGKILL gather
+	ps -o sess --no-headers | sort -u | while read sid; do
```

Comment:
I think, we don't need this comment anymore https://github.com/openshift/oc/blob/a43428189603f12ae190703cb725f66f742fbe52/pkg/cli/admin/mustgather/mustgather.go#L90-L92